Signal/src/util/OWSScreenLockUI.m

```diff
@@ -19,6 +19,10 @@ @interface OWSScreenLockUI ()
 @property (nonatomic) BOOL hasUnlockedScreenLock;
 @property (nonatomic) BOOL isShowingScreenLockUI;
 
+@property (nonatomic, nullable) NSTimer *screenLockUITimer;
```

Comment:
We use a timer to ensure that there's only one "delayed show unlock UI" in flight at a time.